packages/neovim/README.md

```diff
@@ -0,0 +1,90 @@
+# Snakeskin Neovim plugin
+
+## Table of Contents
+
+- [Introduction](#introduction)
+- [Installation](#installation)
+- [Setup](#setup)
+- [Update](#update)
+
+## Introduction
+
+This plugin adds support for the Snakeskin language in Neovim.
+Currently, the following features are supported:
+
+- Syntax highlighting
+- Diagnostic messages (for parsing errors)
+- Go to declaration by include derective
+- Include derective validation for paths exist
+
+Only Jade-like syntax is supported.
+
+## Installation
+
+### [lazy.nvim](https://github.com/folke/lazy.nvim)
+
+```lua
+{
+    "neovim/nvim-lspconfig",
+    opts = {
+        servers = {
+            snakeskin_ls = {}
+        }
+    }
+}
+```
+
+## Setup
+
+### Manual setup
+
+For plugin we need globally pre-installed @snakeskin/cli because plugin run cli command
```

Comment:
```suggestion
For the plugin to work, we need to install `@snakeskin/cli` globally because the plugin runs the LSP server through a CLI subcommand:
```